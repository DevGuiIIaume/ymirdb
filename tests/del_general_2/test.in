set a 1
set b a
set c b
set d 4
list entries
type a
type b
type c
type d
push d a
type d
list entries
forward a
forward d
backward a
backward d
del d
backward a
backward d
set d 4
push a d
type a
forward a
forward b
forward c
forward d
backward a
backward b
backward c
backward d
del d
del a
del b
del c
backward d
del d
del a
del b
backward d
backward a
del a
del d
list entries
bye
