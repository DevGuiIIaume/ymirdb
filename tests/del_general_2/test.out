> ok

> ok

> ok

> ok

> d [4]
c [b]
b [a]
a [1]

> simple

> general

> general

> simple

> ok

> general

> d [a 4]
c [b]
b [a]
a [1]

> nil

> a

> b, c, d

> nil

> ok

> b, c

> no such key

> ok

> ok

> general

> d

> a, d

> a, b, d

> nil

> b, c

> c

> nil

> a, b, c

> not permitted

> not permitted

> not permitted

> ok

> a, b

> not permitted

> not permitted

> ok

> a

> nil

> ok

> ok

> no entries

> bye
